opl/cluster_read.py

```diff
@@ -82,7 +82,10 @@ def measure(self, ri, name, monitoring_query, monitoring_step):
         assert ri.start is not None and ri.end is not None, \
             "We need timerange to approach Prometheus"
         # Get data from Prometheus
-        url = f'{self.host}:{self.port}/api/v1/query_range'
+        if self.host.endswith('.svc'):
```

Comment:
Why not to just specify correct port? Then i should work without this hack I think.

Reply:
Hello Jan, thanks for the feedback. I am using host information available from `oc get routes/prometheus-k8s -n openshift-monitoring` as a prometheus-host and looks like it doesn't accept any port information but instead it rely on the "ports selected by the service this route points to".  

Current defaul hostname: `https://prometheus-k8s.openshift-monitoring.svc`

If try with with port then I would see the following exceptions.
```
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2eecd84640>: Failed to establish a new connection: [Errno 110] Connection timed out
```

Reply:
After sync with @jhutar , I came to understand that this change is not required and I should pass`443` as `--prometheus-port` when working outside the openshift cluster.  Changes are reverted in https://github.com/redhat-performance/opl/pull/62/commits/49ddb40ecfe0f1bd780d1dfa6abd0a2a84af1689